test/native/MODCLUSTER-785/testit.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+source ../includes/script.bash
+
+# check IMG...
+if [ -z ${IMG} ]; then
+  echo "IMG needs to defined, please try"
+  echo "export IMG=quay.io/${USER}/tomcat_mod_cluster"
+  exit 1
+fi
+# check mod_proxy_cluster or mod_cluster
+if [ -z $USE13 ]; then
+  SOURCES=https://github.com/modcluster/mod_proxy_cluster
+  MDBRANCH=main  
+else
+  SOURCES=https://github.com/modcluster/mod_cluster
+  MDBRANCH=1.3.x  
+fi
+
+# first stop any previously running tests.
+stoptomcats
+removetomcats
+
+# and httpd
+docker stop MODCLUSTER-785
+docker rm MODCLUSTER-785
+
+# build httpd + mod_proxy_cluster
+#
+LOCREPO=`git remote -v | grep origin | grep fetch | awk ' { print $2 } ' | awk -F/ ' { print $4 } '`
+REPO="https://raw.githubusercontent.com/modcluster/mod_proxy_cluster/main/test/native/MODCLUSTER-785/mod_proxy_cluster.conf"
+if [ $LOCREPO != "modcluster" ]; then
+  BRANCH=`git branch -v | awk ' { print $2 } '`
+  REPO="https://raw.githubusercontent.com/$LOCREPO/mod_proxy_cluster/${BRANCH}/test/native/MODCLUSTER-785/mod_proxy_cluster.conf"
+fi
+rm -f nohup.out
+nohup docker run --network=host -e HTTPD=https://dlcdn.apache.org/httpd/httpd-2.4.57.tar.gz -e SOURCES=${SOURCES} -e BRANCH=${MDBRANCH} -e CONF=$REPO --name MODCLUSTER-785 quay.io/${USER}/mod_cluster_httpd &
```

Comment:
There is no reason supply `HTTPD` variable when it is the same is the default. Also, wouldn't it be better to use `-d` instead of `&`.?

Reply:
I will fix that.. But it is probably everywhere...

Reply:
I don't get the HTTPD variable part...

Reply:
If you look into https://github.com/modcluster/mod_proxy_cluster/blob/main/test/native/httpd/Dockerfile, there is `ENV HTTPD=https://dlcdn.apache.org/httpd/httpd-2.4.57.tar.gz`, so you don't have to run it with `-e HTTPD=https://dlcdn.apache.org/httpd/httpd-2.4.57.tar.gz`.

Reply:
OK we might need HTTP as an environment variable to test old version of httpd like 2.4.53 ;-)

Reply:
dlcdn.apache.org only has only the latest version normally.

Reply:
I am also thinking of an option to run with a locally install httpd, which is what I need to debug MODCLUSTER-785.

Reply:
If it is due to the maintainer mode, you could try the image we run in CI. https://github.com/modcluster/ci.modcluster.io/tree/main/httpd-dev

Reply:
Actually the -d option is a BAD idea... the waitforhttpd waits for the nohup file...